exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -165,13 +191,47 @@
       checkAnswer() {
         if (this.itemRenderer) {
           const check = this.itemRenderer.scoreInput();
-          this.correct = check.correct;
           this.empty = check.empty;
+          if (!check.empty) {
+            this.complete = check.correct;
+            this.correct = this.hinted || !this.firstAttempt ? false : check.correct;
+            this.$parent.$emit('checkanswer', this.correct, this.complete, this.firstAttempt, this.hinted);
```

Comment:
It's uncommon for a child component to directly reference the parent component. It goes against the composition rule of only communicating with events: https://vuejs.org/guide/components.html#Composing-Components